### DISCUSSION TITLE
Welcome to deepblast Discussions!

### DISCUSSION THREAD

**Discussion by mortonjt** [flatironinstitute/deepblast#126] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

If you have questions on how to use DeepBLAST and TM-vec, feel free to raise questions.

**Comment by elemily** [comment]
Can't load tokenizer for 'Rostlab/prot_t5_xl_uniref50'. If you were trying to load it from 'https://huggingface.co/models', make sure you don't have a local directory with the same name. Otherwise, make sure 'Rostlab/prot_t5_xl_uniref50' is the correct path to a directory containing all relevant files for a T5Tokenizer tokenizer.
----这个报错怎么解决啊，我有这个目录呀

**Reply by NRX-044** [reply]
I solved it by putting my script and local ProTrans repository under the same directory. See also: https://stackoverflow.com/questions/64001128/load-a-pre-trained-model-from-disk-with-huggingface-transformers

**Comment by mortonjt** [comment]
Hi could you provide the exact error?

You could also try not specifying the protrans model to see if the error
goes away. This will automatically download the protrans model
